# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/html/dom/idlharness.worker.js.ini
# Commit: 40fa8e4b06c8
# Full Hash: 40fa8e4b06c85e9e241da6e56b62a0b2bde7427c
# Author: Andrew Osmond <aosmond@mozilla.com>
# Date: 2024-03-28 17:08:58
# Regressor Bug: 1887729
# File Overlap Count: 2
# Description:
#   Bug 1887729 - Implement context lost/restored support for CanvasRenderingContext2D. r=webidl,gfx-reviewers,smaug,lsalzman
#   
#   Remote canvas can run in the GPU process, and if the GPU process
#   crashes, we need to notify the application using canvas. Historically we
#   just failed, and the application may have been able to continue drawing
# ==============================================================================

diff -r 469884ac63bc -r 40fa8e4b06c8 testing/web-platform/meta/html/dom/idlharness.worker.js.ini
--- a/testing/web-platform/meta/html/dom/idlharness.worker.js.ini	Thu Mar 28 01:16:16 2024 +0000
+++ b/testing/web-platform/meta/html/dom/idlharness.worker.js.ini	Thu Mar 28 01:51:23 2024 +0000
@@ -5,9 +5,6 @@
   [ImageData interface: attribute colorSpace]
     expected: FAIL
 
-  [OffscreenCanvasRenderingContext2D interface: operation isContextLost()]
-    expected: FAIL
-
   [PromiseRejectionEvent interface: attribute promise]
     expected: FAIL
 